Makefile

```diff
@@ -155,7 +157,7 @@ copy_node_js:
 ifeq ($(IN_DOCKER),)
 	$(warning Command is designed to be run in the container)
 endif
-	for dest in $(NODE_LIBS_JS) ; do cp $(NODE_MODULES)$$dest $(STATIC_JS) ; done
+	for dest in $(NODE_LIBS_JS) ; do cp -r $(NODE_MODULES)$$dest $(STATIC_JS) ; done
```

Comment:
Would it be possible to make this explict rather than copying entire directories? I'd have concerns that this way we don't really know what we're putting on the CDN.